x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_signal.ts

```diff
@@ -80,7 +80,7 @@ export const removeClashes = (doc: BaseSignalHit): BaseSignalHit => {
 export const buildSignal = (
   docs: BaseSignalHit[],
   rule: RulesSchema,
-  owner: typeof SERVER_APP_ID
+  owner: string // typeof SERVER_APP_ID
```

Comment:
increasing scope of this type to `string` since it is possible something else could be the consumer of a siem rule.